scss/_patterns_links.scss

```diff
@@ -45,6 +46,10 @@
         color: darken($color-light, 10%);
       }
     }
+
+    & .p-link--no-underline:hover {
```

Comment:
Is this a new pattern?

Reply:
Same as above, wanted to make the card wrappable in an <a> but keep text from getting underlines. Reverted.